### PR TITLE
Review sec

### PR DESCRIPTION
**Pull request by Vedant1206**
Fixed bugs

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -815,7 +817,6 @@
     "unicode-match-property-value-ecmascript": "^2.1.0",
     "unicode-property-aliases-ecmascript": "^2.1.0",
     "unique-string": "^2.0.0",
-    "universal-cookie": "^7.1.0",
```
Comment (KatieCodess) [inline client/package.json:818]: Why did you remove this?

**Review by ChristopherE0 — APPROVED** [review]
Looks good. One question in Discord.

**Review by KatieCodess — COMMENTED** [review]
Why did you remove universal cookies from package.json

File: client/package.json (hunk 1 above)
Comment (Vedant1206) [reply client/package.json:818]: I by mistake have done it but it was working from my end

Comment (Vedant1206) [reply client/package.json:818]: Ive added that! thanks
